ambry-clustermap/src/test/java/com/github/ambry/clustermap/MockHelixAdmin.java

```diff
@@ -579,6 +600,16 @@ public void dropResource(String clusterName, String resourceName) {
     throw new IllegalStateException("Not implemented");
   }
 
+  @Override
+  public void addCloudConfig(String clusterName, CloudConfig cloudConfig) {
+
```

Comment:
Curious why you throw IllegalStateException in the other methods but not these?

Reply:
My bad, will add it to these methods.